calc.js

```diff
@@ -6,19 +6,25 @@ const assert = require('./assert');
 
 module.exports = (op, a, b) => {
   assert(!!op, 'op must be required');
-  assert(!!a, 'a must be required');
-  assert(!!b, 'b must be required');
 
   if (op === '+') {
+    if (!a) throw new Error('a must be required')
+    if (!b) throw new Error('b must be required')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ellyofreitas/node-code-climate/pullRequest?prid=7918638)
```suggestion
    if (!b) throw new Error("b must be required")
```